CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.9] - 2023-06-20
+
+### Added
+
+- CASM-4350: Implement a proto type of Split config maps, to store cray-product-catalog data...
```

Comment:
Can you elaborate more on what this change means? I think it would be difficult to understand for someone who wasn't also looking at the PR code changes. It doesn't have to be super detailed -- you can assume that the reader is familiar with how the product catalog currently operates and the existing Python code.